examples/interdomain/README.md

```diff
@@ -36,49 +36,57 @@ export KUBECONFIG=$KUBECONFIG1
 kubectl create ns nsm-system
 ```
 
-Register `nsm-system` namespace in spire:
-
-```bash
-kubectl exec -n spire spire-server-0 -- \
-/opt/spire/bin/spire-server entry create \
--spiffeID spiffe://nsm.cluster1/ns/nsm-system/sa/default \
--parentID spiffe://nsm.cluster1/ns/spire/sa/spire-agent \
--selector k8s:ns:nsm-system \
--selector k8s:sa:default \
--federatesWith spiffe://nsm.cluster2 \
--federatesWith spiffe://nsm.cluster3
-```
-
-Register `registry-k8s-sa` in spire:
-
-```bash
-kubectl exec -n spire spire-server-0 -- \
-/opt/spire/bin/spire-server entry create \
--spiffeID spiffe://nsm.cluster1/ns/nsm-system/sa/registry-k8s-sa \
--parentID spiffe://nsm.cluster1/ns/spire/sa/spire-agent \
--selector k8s:ns:nsm-system \
--selector k8s:sa:registry-k8s-sa \
--federatesWith spiffe://nsm.cluster2 \
--federatesWith spiffe://nsm.cluster3
-```
-
-Register `nsmgr-proxy-sa` in spire:
-
-```bash
-kubectl exec -n spire spire-server-0 -- \
-/opt/spire/bin/spire-server entry create \
--spiffeID spiffe://nsm.cluster1/ns/nsm-system/sa/nsmgr-proxy-sa \
--parentID spiffe://nsm.cluster1/ns/spire/sa/spire-agent \
--selector k8s:ns:nsm-system \
--selector k8s:sa:nsmgr-proxy-sa \
--federatesWith spiffe://nsm.cluster2 \
--federatesWith spiffe://nsm.cluster3
+Create nsmgr-proxy patch:
+```bash
+cat > patch-nsmgr-proxy.yaml <<EOF
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nsmgr-proxy
+spec:
+  template:
+    metadata:
+      annotations:
+        spiffe.io/federatesWith: nsm.cluster2,nsm.cluster3
+EOF
```

Comment:
This can be a static file. Please put it into /cluster1, /cluster2 and so on

Reply:
ok, done